blog/_src/posts/2017-10-11-tutorial-creating-a-package.md

```diff
@@ -0,0 +1,297 @@
+    Title: Tutorial: Creating a Package
+    Date: 2017-10-11T15:46:21
+    Tags: 
+
+*posted by Stephen Chang*
+
+This post is a summary of a tutorial presented at [RacketCon
+2017](http://con.racket-lang.org/2017/).
+
+It describes how to create a package starting from a single Racket file.
+
+Specifically, this post explains how to:
+
+- use `raco pkg install` to install a package,
+- create an `info.rkt` file containing package metadata, and
+- add a package to
+  [Racket's official package catalog](https://pkgs.racket-lang.org).
+
+<!-- more -->
+
+## Packages vs Collections
+
+Before getting started, we first clarify some terminology.
+
+In Racket, a library is just a [module][doc-module] (typically in its own file)
+that exports some bindings. A programmer may import such a module using a
+(relative or absolute) filepath:
+
+```racket
+(require "path/to/filename.rkt")
+```
+
+A Racket [**collection**][doc-collect] is a "hierarchical group of modules"
```

Comment:
You may want to link to this post, which discusses this topic in more depth:
http://blog.racket-lang.org/2015/08/modules-packages-and-collections.html